content/en/02-installing-pixie/03-install-guides/02-self-hosted-pixie/index.md

```diff
@@ -5,7 +5,7 @@ metaDescription: "Getting started guide to setup 100% self-hosted Pixie"
 order: 2
 ---
 
-Get Pixie fully managed with [Pixie Community Cloud](/installing-pixie/install-guides/community-cloud-for-pixie) (free forever) or run on your own infrastructure with the following self-managed option.
+Get Pixie fully managed with one of the [Hosted Pixie](/installing-pixie/install-guides/hosted-pixie) or run on your own infrastructure with the following self-managed option.
```

Comment:
Add `offerings`: "managed with one of the `[Hosted Pixie offerings]`".

Reply:
Fixed in latest commit.